ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/patch-panel.j2

```diff
@@ -0,0 +1,26 @@
+{% if patch_panel is arista.avd.defined %}
+!
+patch panel
+{%     for patch in patch_panel.patches | arista.avd.natural_sort %}
+{%         if patch.name is arista.avd.defined %}
+   patch {{ patch.name }}
+{%             if patch.enabled is arista.avd.defined(false) %}
+      shutdown
+{%             endif %}
+{%             for connector in patch.connectors | arista.avd.natural_sort %}
+{%                 if connector.id is arista.avd.defined %}
+{%                     if connector.interface is arista.avd.defined %}
+{%                         set connector_cli = "connector " ~ connector.id ~ " interface " ~ connector.interface %}
+{%                         if connector.dot1q_vlan is arista.avd.defined %}
+{%                             set connector_cli = connector_cli ~ " dot1q vlan " ~ connector.dot1q_vlan %}
+{%                         endif %}
+      {{ connector_cli }}
+{%                     elif connector.pseudowire is arista.avd.defined %}
```

Comment:
this section should be part of building `connector_cli` as above.

Reply:
Fixed.